org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/StructureDefinitionRenderer.java

```diff
@@ -1289,23 +1289,23 @@ private List<ElementDefinitionMappingComponent> getMap(ElementDefinition e, Stri
 
   public String header() throws Exception {
     StringBuilder b = new StringBuilder();
-    b.append("<p>\r\n");
+    b.append("<p class=\"profile-url-label\">\r\n");
     b.append("The official URL for this profile is:" + "\r\n");
     b.append("</p>\r\n");
-    b.append("<pre>" + sd.getUrl() + "</pre>\r\n");
-    b.append("<p>\r\n");
+    b.append("<pre class=\"profile-url\">" + sd.getUrl() + "</pre>\r\n");
+    b.append("<div class=\"profile-description\">\r\n");
```

Comment:
changed `<p>` to `<div>` because the markdown can contain `<p>` which I don't think should be surrounded by outer `<p>`

Reply:
if it doesn't contain a paragraph, does this change the way it displays?

Reply:
It depends how the stylesheet handling of `p` but generally paragraph don't have any margin etc.